doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -327,7 +327,7 @@ julia> 2*Float16(4.)
 Float16(8.0)
 ```
 
-The underscore `_` can be used as digit separator:
```

Comment:
I think the previous way was better (see e.g. https://github.com/dotnet/roslyn/issues/216)